mulhome_scripts/delete_all_waves.py

```diff
@@ -93,42 +98,46 @@ def delete_all_waves(app_name):
         resp = api.list_namespaced_replica_set(label_selector = label,namespace=namespace)
         # if a replicaset exist, delete it
         # pprint(resp)
-        # print resp.items[0].metadata.namespace
+        # logging.debug resp.items[0].metadata.namespace
         for i in resp.items:
             if i.metadata.namespace == namespace:
-                del_resp_1 = api.delete_namespaced_replica_set(i.metadata.name, namespace, body)
-                print("Relicaset '%s' Deleted. status='%s'" % (pod_name, str(del_resp_1.status)))
+                # del_resp_1 = api.delete_namespaced_replica_set(i.metadata.name, namespace, body)
+                del_resp_1 = api.delete_namespaced_replica_set(i.metadata.name, namespace)
+
+                logging.debug("Relicaset '%s' Deleted. status='%s'" % (pod_name, str(del_resp_1.status)))
 
         # Check if there is a pod still running by using the label
         resp = None
         api_2 = client.CoreV1Api()
         resp = api_2.list_namespaced_pod(namespace, label_selector = label)
         # if a pod is running just delete it
         if resp.items:
-            del_resp_2 = api_2.delete_namespaced_pod(resp.items[0].metadata.name, namespace, body)
-            print("Pod Deleted. status='%s'" % str(del_resp_2.status))
+            # del_resp_2 = api_2.delete_namespaced_pod(resp.items[0].metadata.name, namespace, body)
+            del_resp_2 = api_2.delete_namespaced_pod(resp.items[0].metadata.name, namespace)
+
+            logging.debug("Pod Deleted. status='%s'" % str(del_resp_2.status))
 
         # Check if there is a service running by name = key
         resp = None
         api_2 = client.CoreV1Api()
         try:
             resp = api_2.read_namespaced_service(pod_name, namespace)
         except ApiException as e:
-            print("Exception Occurred")
+            logging.debug("Exception Occurred")
         # if a service is running, kill it
         if resp:
-            #del_resp_2 = api_2.delete_namespaced_service(pod_name, namespace)
-            del_resp_2 = api_2.delete_namespaced_service(pod_name, namespace,body)
-            print("Service Deleted. status='%s'" % str(del_resp_2.status))
+            del_resp_2 = api_2.delete_namespaced_service(pod_name, namespace)
+            # del_resp_2 = api_2.delete_namespaced_service(pod_name, namespace,body)
```

Comment:
Lots of commented code to delete. This seems to be a recurring issue across these mulhome scripts, so maybe spend time deleting them so these files are more readable. They seem to not be touched often by development so this seems safe to do?

Reply:
Same reasons above for all the scripts in mulhome_scripts.